.circleci/config.yml

```diff
@@ -1,164 +1,113 @@
 version: 2.1
 
-parameters:
-  preview:
-    type: boolean
-    default: true
-  3-10:
-    type: boolean
-    default: true
-  3-11:
-    type: boolean
-    default: true
+# orbs:
+#   coveralls: coveralls/coveralls@2.2.1
+
+workflows:
+  ci:
+    jobs:
+      - lint
+      - test:
+          matrix:
+            parameters:
+              # TODO: Revisit why pyenv doesn't recognize 3.12
+              python_version: ["3.8", "3.9", "3.10", "3.11"] # "3.12"
```

Comment:
For some reason `pyenv 2.3.37` does not recognize `3.12` when attempting to do `pyenv install 3.12`...

However, `pyenv install 3.12` worked fine locally with `pyenv 2.3.31` 🤔